src/lib/generatedMarkdown.js

```diff
@@ -178,7 +178,7 @@ export async function generateReportingStructure(starting_role) {
   let pageContent = ''
   for (const [teamMemberName, teamMember] of Object.entries(teamMembers)) {
     if (teamMember.manager_role_slug === starting_role) {
-      pageContent += `- [${String(teamMember.name)}](../../../../company/team/index.md#${String(
+      pageContent += `- [${String(teamMember.name)}](/company/team/index.md#${String(
```

Comment:
To match our other [bio link logic](https://github.com/sourcegraph/handbook/blob/19dff2e1a37616251ae8a025937ec3e7a2fe6a4e/src/lib/generatedMarkdown.js#L18)